docs/code-and-learn-agenda.md

```diff
@@ -0,0 +1,107 @@
+today's Code and Learn we are going to go through some of those ways to get involved and help you all make meaningful contributions to the project.
+
+The project is composed of 43 repos in 3 github orgs:
+
+- `expressjs`: The primary org where the `express` repo lives. The other repos in this org are all directly part of `express` or dependent on `express` api's
+- `pillarjs`: This org is where generic but high level server components are hosted. Things like the `router` and `path-to-regexp` which can be used outside of `express` but are
+  still high level.
+- `jshttp`:  This org is where lower level `http` and protocol level packages are hosted. Things like `cookie` and `mime-db`.
+
+Because we are so spread out, we use a repo in the `expressjs` org called `discussions` to have cross-cutting or project wide discussions. Often when deciding what to work on, this
+is the best place to start. More on that to come.
+
+Lets talk a little bit about the project governance and leadership. We have a few roles which are great ways to start onboarding to the project:
+
+- The triage team: this team focuses on helping answer the many questions which come into the project issue tracker, as well as how to best organize answers and update docs
+- The security working group: this group is in charge of helping us define and maintain our security model, docs, and reporting
+
+Additionally we have roles for folks who are making technical contributions:
+
+- Contributor: these are folks who have made a few contributions to code and are given commit access to land PRs and their own work
+- Repo Captains: these are folks who where contributors on a given repo but have demonstrated strong judgement and reliability who are also given publish permissions and more
+  responsibility within the given repo
+
+And finally we have the leadership group, which is the Technical Committee or as we call it, the TC. This group oversees the larger project direction and is there to help resolve conflicts or make
+important decisions.
+
+## Questions
+
+Does anyone have questions on the structure of the project?
+
+## Project Goals (10 - 15 min)
+
+Because the project is made up of individual volunteers with many different skills, we attempt to organize the priorities and work with a distributed consensus model. This means
+the goals are yours to set. That said, currently because the project is undergoing a revitalization effort, we are maintaining a central list of goals in the `discussions`
+repository.
+
+### General Goals
+
+https://github.com/expressjs/discussions
+
+### Express 5 Goals
+
+https://github.com/expressjs/discussions/issues/266
+
+### Express 6 Goals
+
+https://github.com/expressjs/discussions/issues/267
+
+### Questions
+
+Anyone have questions before we start contributing?
+
+## Lets Start Contributing (1-1.5h)
+
+We picked out three specific goals the project has which we think will make simple yet very helpful contributions today:
+
```

Comment:
```suggestion

0. Join the team offline and follow the discussions in https://openjsf.org/collaboration
  - Join the Slack channel `#express`
  - Follow the calendar entries   
```

Reply:
I think that this opens the collaboration for a long term, as the people can keep engaging with us after the event :)